src/PgRvn.Server/SqlQuery.cs

```diff
@@ -0,0 +1,422 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO.Pipelines;
+using System.Linq;
+using System.Reflection.Metadata;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Raven.Client.Documents.Linq.Indexing;
+using Raven.Client.Documents.Queries;
+using Sparrow.Json;
+using TSQL;
+using TSQL.Statements;
+using TSQL.Tokens;
+
+
+namespace PgRvn.Server
+{
+    public class SqlQuery : PgQuery
```

Comment:
what is this for?

Reply:
For hardcoded queries - renamed it to HardcodedQuery